streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedCachingPersistentWindowStoreTest.java

```diff
@@ -124,7 +128,8 @@ public static Collection<Object[]> data() {
     public void setUp() {
         baseKeySchema = new TimeFirstWindowKeySchema();
         bytesStore = new RocksDBTimeOrderedWindowSegmentedBytesStore("test", "metrics-scope", 100, SEGMENT_INTERVAL, hasIndex);
-        underlyingStore = new RocksDBTimeOrderedWindowStore(bytesStore, false, WINDOW_SIZE);
+        underlyingStore = spy(new RocksDBTimeOrderedWindowStore(bytesStore,
```

Comment:
`underlyingStore` is reset only in the close tests (even that is not required, see other comment). In the close tests, the `underlyingStore` member of this class is replaced in the function `setUpCloseTests()` and hence, we don't need to spy this.